src/main/java/com/google/sps/servlets/CreateGameServlet.java

```diff
@@ -43,35 +45,86 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/index.html");
     }
 
+    /**
+    * Retrieves the game's name.
+    * @param request the HttpServletRequest of the doPost.
+    * @return a String representing the name/title of the game.
+    */
     private String getName(HttpServletRequest request) {
         return request.getParameter("gameName");
     }
 
+    /**
+    * Retrieves the game's description.
+    * @param request the HttpServletRequest of the doPost.
+    * @return a String representing the description of the game.
+    */
     private String getDescription(HttpServletRequest request) {
         return request.getParameter("gameDescription");
     }
 
     // Below: TODO once game creation form is done
+    /**
+    * Retrieves the number of stages in the game.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an int representing the number of stages.
+    */
     private int getNumStages(HttpServletRequest request) {
         return -1;
     }
 
+    /**
+    * Retrieves the starting location of each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is a Coordinates representing
+    *         the starting location of the ith stage.
+    * @see com.google.sps.data.Coordinates
+    */
 	private ArrayList<Coordinates> getStageSpawnLocations(HttpServletRequest request) {
         return null;
     }
 
+    /**
+    * Retrieves the starting hint for each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is a String representing
+    *         the starting hint for the ith stage.
+    */
     private ArrayList<String> getStageStarterHints(HttpServletRequest request) {
         return null;
     }
 
+    /**
+    * Retrieves the number of hints (excluding the starter hint) in each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is an int representing
+    *         the number of hints in the ith stage.
+    */
     private ArrayList<Integer> getNumHints(HttpServletRequest request) {
         return null;
     }
 
+    /**
+    * Retrieves the coordinates of each hint in each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is a list of
+    *         Coordinates of the hints on the ith stage. The jth
+    *         element of this list represents the Coordinates of
+    *         the jth hint of the ith stage.
+    * @see com.google.sps.data.Coordinates
+    */
     private ArrayList<ArrayList<Coordinates>> getHintCoordinates(HttpServletRequest request) {
         return null;
     }
 
+    /**
+    * Retrieves the text of each hint in each stage.
+    * @param request the HttpServletRequest of the doPost.
+    * @return an ArrayList, where the ith element is a list of
```

Comment:
For this one and the one above, could you possibly make it more clear what the "ith" and "jth" mean.

Reply:
Done, thanks.